ml-agents/mlagents/trainers/components/bc/module_torch.py

```diff
@@ -0,0 +1,192 @@
+from typing import Dict, Any
+import numpy as np
+import torch
+
+from mlagents.trainers.policy.torch_policy import TorchPolicy
+from mlagents.trainers.demo_loader import demo_to_buffer
+from mlagents.trainers.settings import BehavioralCloningSettings
+from mlagents.trainers.torch.utils import ModelUtils
+
+
+class TorchBCModule:
```

Comment:
This will probably be renamed after we deprecate TF. 
I would prefer calling this class `BCModule` but put it in a `torch` subfolder (like what @ervteng  is doing with the `network.py` classes)

Reply:
I agree. What of your reward providers?